guide/14-deep-learning/unsupervised-ml-model-guide.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>[('raster.tif',MinMaxScaler())]&nbsp;</p><p>or [(raster.name, MinMaxScaler())] -&gt; raster is the Raster object</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>